statsmodels/datasets/tests/test_utils.py

```diff
@@ -1,18 +1,22 @@
 import os
-from ssl import SSLError
 from socket import timeout
+
 from urllib.error import HTTPError, URLError
 
 import numpy as np
 from numpy.testing import assert_, assert_array_equal
 import pytest
 
+from statsmodels.compat.python import PYTHON_IMPL_WASM
 from statsmodels.datasets import get_rdataset, webuse, check_internet, utils
 
-cur_dir = os.path.dirname(os.path.abspath(__file__))
 
-IGNORED_EXCEPTIONS = (HTTPError, URLError, SSLError,  UnicodeEncodeError,
-                      timeout)
+CUR_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
`cur_dir` was being used multiple times in this file, so I had marked this as a constant as well in f20d90bb00634e6f1369b1ac655e1ef91936c5ad.